content/vault/v1.21.x (rc)/content/docs/troubleshoot/tune-lease-ttl.mdx

```diff
@@ -185,6 +189,28 @@ The following command lists all available parameters that you can tune.
 $ vault secrets tune -h
 ```
 
+</Tab>
+<Tab heading="GUI" group="gui">
+
+> **Note:** Vault 1.21 supports configuration updates (tune) for mounted **keymgmt** secrets engine in the GUI. Support for other engines may be added in future releases.
```

Comment:
If we plan to use this note on multiple pages, we should move it to a partial so it's easier to manage over time.